org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/navigation/XbaseHyperLinkHelper.java

```diff
@@ -155,6 +171,30 @@ public void createHyperlinksByOffset(final XtextResource resource, final int off
 		}
 	}
 
+	/**
+	 * Creates hyperlinks on types referenced in javadoc comments.
+	 * @since 2.16
```

Comment:
Small type remark, add an empty line between the JavaDoc description and the `@since`